billing/views.py

```diff
@@ -36,15 +38,12 @@ def invoice_payment_succeeded(self, invoice: stripe.Invoice) -> None:
                 stripe_subscription_id=invoice.subscription,
             ),
         )
-        owner = Owner.objects.get(
+        updated: int = Owner.objects.filter(
```

Comment:
if this returns None for the first filter will the update throw an error?

Reply:
`.filter()` will always return a QuerySet. It can be an empty queryset, but it never returns None. To be certain though, I did test this and there was no error thrown 🙂

Reply:
This is a really good logic update - instead of 2 db hits (1 `.get()`, then 1 `.save()`) you've reduced it to 1 hit that is also more efficient, and you correctly recognized that we aren't doing anything with the `owner` object, we don't need to load that into memory. 
Since we're scrutinizing this section, what's the reasoning behind `_log_updated`? Why are we logging a count of objects updated with no other info, when would this be a useful metric? I understand logging for the error state (`Could not find customer`) but it's also not helpful if it contains no info about the operation that was attempted. What do y'all think?

Reply:
@nora-codecov thanks for the comment! Re `_log_updated` see https://github.com/codecov/codecov-api/pull/832#discussion_r1771780317